README.md

```diff
@@ -3,51 +3,62 @@
 Add footnotes functionality to your Wagtail project.
 
 ## Usage
- - Add the app to `INSTALLED_APPS`:
-   ```python
-   INSTALLED_APPS = [
-       ...
-       "wagtail_footnotes",
-       ...
-   ]
-   ```
- - Add the footnotes `urls.py` to your project's `urls.py`:
-   ```python
-   from wagtail_footnotes import urls as footnotes_urls
-   urlpatterns = [
-       ...
-       path("footnotes/", include(footnotes_urls)),
-       ...
-   ]
-   ```
-   Note: The URL has to be defined as above as it is currently hardcoded in the Javascript.
- - Update your page models to show the footnotes field:
-   ```python
-   class InformationPage(BasePage):
+
```

Comment:
Look at some of the changes made here for inclusion. https://github.com/torchbox/wagtail-footnotes/pull/37

Reply:
I decided not to use #37

The code example isn't catered for in the package.